common/app/views/support/Commercial.scala

```diff
@@ -15,7 +15,7 @@ object Commercial {
   def isAdFree(request: RequestHeader): Boolean = {
     try {
       request.headers.get("X-GU-Commercial-Ad-Free").exists(_.toLowerCase == "true") ||
-        request.cookies.get("GU_AF1").exists(_.value.toInt > 0)
+        request.cookies.get("GU_AF1").exists(_.value.toLong > 0)
```

Comment:
This was the ad-free expiry in my cookie:
![image](https://user-images.githubusercontent.com/3072877/42702914-cf6d11f6-86c2-11e8-9fdb-bae930ca3cb9.png)
and
![image](https://user-images.githubusercontent.com/3072877/42702881-b467ac40-86c2-11e8-967b-0c281be48eda.png)

Reply:
Good find! I haven't run into that issue yet.